Hippo.Web/Controllers/SoftwareController.cs

```diff
@@ -0,0 +1,84 @@
+using Hippo.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Hippo.Core.Models;
+using Hippo.Email.Models;
+
+namespace Hippo.Web.Controllers;
+
+[Authorize]
+public class SoftwareController : SuperController
+{
+    private readonly IUserService _userService;
+    private readonly IHistoryService _historyService;
+    private readonly IEmailService _emailService;
+    private readonly INotificationService _notificationService;
+
+    public SoftwareController(IUserService userService, IHistoryService historyService,
+        IEmailService emailService, INotificationService notificationService)
+    {
+        _userService = userService;
+        _historyService = historyService;
+        _emailService = emailService;
+        _notificationService = notificationService;
+    }
+
+    [HttpPost]
+    public async Task<ActionResult> RequestInstall([FromBody] SoftwareRequestModel softwareRequestModel)
+    {
+        if (!ModelState.IsValid)
+        {
+            return BadRequest(ModelState);
+        }
+
+        var currentUser = await _userService.GetCurrentUser();
```

Comment:
Might not be good enough (at least if you use the name), I think if it is in the DB we don't pull from IAM? And only IAM will have the latest name.